annotation-server/src/common/google-sheets.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { Test, TestingModule } from '@nestjs/testing';
+
+import { fetchSpreadsheetCells } from './google-sheets';
+
+describe('Helper to fetch spreadsheet', () => {
+    let configService: ConfigService;
+
+    beforeEach(async () => {
+        const modelFixture: TestingModule = await Test.createTestingModule({
+            imports: [
+                ConfigModule.forRoot({ envFilePath: ['.env', 'test/.env'] }),
```

Comment:
If I see this correctly the paths are in the wrong order here and this prioritises the `.env` over the `test/.env` as opposed to the normal module. Is this wanted?

Reply:
No, my bad. Will fix